src/components/structures/RoomView.js

```diff
@@ -516,21 +518,26 @@ module.exports = React.createClass({
         }
     },
 
-    _updateTabCompleteList: new rate_limited_func(function() {
```

Comment:
@dbkr Not entirely sure, but afaik, this shouldn't cause problems since the rate limiting was earlier in place when we had to copy over memberlist + roomlist + commands + emoji into an array on each update. We don't do that anymore.